INSTALL.md

```diff
@@ -0,0 +1,68 @@
+# Install Tomateos
+### Required dependencies
+
+Required dependencies trough Packet-Managers/Manual installation:
+* Ruby >= 2.3.3
+* Ruby-Gems
```

Comment:
As far as i saw, at least Fedora packaged it seperatly: https://admin.fedoraproject.org/pkgdb/package/rpms/rubygems/